src/SFA.DAS.ApplyService.Web/Views/Application/PagesWithSections.cshtml

```diff
@@ -1,15 +1,20 @@
-﻿@model SFA.DAS.ApplyService.Domain.Entities.ApplicationSection
+﻿@using SFA.DAS.ApplyService.Web.ViewModels.Roatp
+@model SFA.DAS.ApplyService.Domain.Entities.ApplicationSection
 
 @{
     ViewBag.Title = @Model.Title;
     Layout = "_Layout";
     var pageTitle = "";
     var isSamePage = false;
     var nextPageTitle = "";
-    var pagesToCheck = Model.QnAData.Pages.Where(p => p.Active && !p.NotRequired).ToList();
+    var pagesToCheck = Model.QnAData.Pages.Where(p => p.Active && !p.NotRequired && p.DisplayType!=null && p.DisplayType == "PagesWithSections").ToList();
```

Comment:
Possibly write this as a method on the view model, rather than logic in the view

Reply:
A lambda property on the viewmodel would work :)